### PR TITLE
BoxView Handlers

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

**BoxView** Handlers for:
- Android
- iOS
- macOS
- Windows

boxviewhandler

Implement the BoxView control using Maui **Graphics**.

### PR Checklist ###

- [x] Targets the correct branch 
- [x] Tests are passing (or failures are unrelated)
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might affect accessibility?
- No

### CODE REVIEW

File: src/Core/src/Core/IBoxView.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+	/// <summary>
+	/// BoxView renders a simple rectangle of a specified width, height, and color. 
+	/// </summary>
+	public interface IBoxView : IView
+	{
+		/// <summary>
+		/// Gets the color which will fill the rectangle.
+		/// </summary>
+		Color Color { get; }
```
Comment (rmarinho) [inline src/Core/src/Core/IBoxView.cs:13]: Should this be a Paint ? or a Brush ?

Comment (AmrAlSayed0) [reply src/Core/src/Core/IBoxView.cs:13]: Also, there is no need for a special `Color` property, at least in core. `BackgroundColor/Brush` would do. The Controls can map to that property to stay backward compatible.

Comment (rmarinho) [reply src/Core/src/Core/IBoxView.cs:13]: created a ussye ti follow up on this https://github.com/dotnet/maui/issues/2365

**Review by rmarinho — APPROVED** [review]
Seems to work fine , we can improve this by using `Paint` https://github.com/dotnet/maui/issues/2365

**Pull request merged**